.gitignore

```diff
@@ -1,4 +1,3 @@
 vendor
 .php-cs-fixer.cache
-composer.lock
```

Comment:
k10r/codestyle is only to be used as a dependency. Do we need the lockfile in here for some testing scenario?

Reply:
Good point, I checked it, and it isn't necessary 🙂